recipe/meta.yaml

```diff
@@ -1,40 +1,45 @@
 {% set name = "ipython_memory_usage" %}
-{% set version = "1.1" %}
+{% set version = "1.2" %}
+{% set sha256 = "1704ada665410f0443052efc108f4c8819c60bb7b0d6e7b253a098f0a66284d0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
+  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 1f4697210257f853fea74de0cf3fae60a32e550e578bac6f46de9b40c550422b
+  sha256: {{ sha256 }}
 
 build:
   noarch: python
   number: 0
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
 
 requirements:
-  host:
+  build:
     - python
     - pip
+
   run:
     - python
-    - memory_profiler
     - ipython
 
 test:
-  imports:
```

Comment:
Why no import test anymore?

Reply:
From the gitter history it seems this is ok but needs a comment on why.

Reply:
The package is for IPython and in order to succeed tests need to be run with this module.

This has been replaced by importing the library through the python executable.

commands:
    - python -m IPython "import ipython_memory_usage"

Reply:
Hi @xhochy as @ps-git notes we need IPython for this to run, I've posted an example on this issue so that in the future I'll issue a useful exception message if it is imported from outside the IPython environment: https://github.com/ianozsvald/ipython_memory_usage/issues/24